content/Reducing_Power_Consumption.md

```diff
@@ -26,30 +27,30 @@ When waiting for a vblank event, this would be a BAD example:
 ```
 
 A better example would be a procedure as shown below. In this case the
-vblank interrupt must be enabled, and your vblank interrupt procedure
+v-blank interrupt must be enabled, and your v-blank interrupt handler
 must set vblank_flag to a non-zero value.
 
 ```
   ld   hl,vblank_flag  ;hl=pointer to vblank_flag
```

Comment:
This snippet is actually vulnerable to a TOCTTOU, if the VBlank interrupt occurs after the `cp [hl]`. Should that be noted?